src/components/Input.tsx

```diff
@@ -40,6 +57,7 @@ export const Input = React.forwardRef<TextInput, InputProps>((props, ref) => {
     loadingPosition = 'right',
     iconRight,
     iconLeft,
+    labelStyle = {},
```

Comment:
do we need default value here?

Reply:
I didn't want to send an "undefined" to the array of styles, therefore this default value is there.

Reply:
You can send optional styles, there is no need to have default object, I suppose

Reply:
label is taken this 
<Label 
styles = [styles.baseLabel, props.labelStyle ]
... />
I added the default object because I dont want to set an "undefined" inside the array. 

If you think its a problem I can delete it.